examples/ice_advected_on_coastline.jl

```diff
@@ -0,0 +1,164 @@
+
+#####
+##### Setup atmospheric and oceanic forcing
+#####
+
+# Atmosphere - sea ice stress
+Ua = XFaceField(grid)
+τᵤ = Field(ρₐ * Cᴰ * Ua^2)
+set!(Ua, 𝓋ₐ)
+compute!(τᵤ)
+Oceananigans.BoundaryConditions.fill_halo_regions!(τᵤ)
+τᵥ = 0.0
+
+#####
+##### Ocean stress (a zero-velocity ocean with a drag)
+#####
+
+τₒ = SemiImplicitStress()
+
+#####
+##### Numerical details
+#####
+
+# We use an elasto-visco-plastic rheology and WENO seventh order 
+# for advection of h and ℵ
+dynamics = SeaIceMomentumEquation(grid; 
+                                  top_momentum_stress = (u=τᵤ, v=τᵥ),
+                                  bottom_momentum_stress = τₒ, 
+                                  rheology = ElastoViscoPlasticRheology(min_substeps=150, 
+                                                                        max_substeps=150,
+                                                                        minimum_plastic_stress=1e-10),
+                                  solver = SplitExplicitSolver(substeps=150))
+
+u_bcs = FieldBoundaryConditions(top = nothing, bottom = nothing,
+                                north = ValueBoundaryCondition(0),
+                                south = ValueBoundaryCondition(0))
+
+#Define the model! 
+model = SeaIceModel(grid; 
+                    advection = WENO(order=7),
+                    dynamics = dynamics,
+                    boundary_conditions = (; u=u_bcs),
+                    ice_thermodynamics = nothing)
+
+# We start with a concentration of ℵ = 1 everywhere
+set!(model, h = 1)
+set!(model, ℵ = 1)
+
+#####
+##### Setup the simulation
+#####
+
+# run the model for 10 day
+simulation = Simulation(model, Δt = 2minutes, stop_time=2days) 
+
+# Container to hold the data
+htimeseries = []
+ℵtimeseries = []
+utimeseries = []
+vtimeseries = []
+
+## Callback function to collect the data from the `sim`ulation
+function accumulate_timeseries(sim)
+    h = sim.model.ice_thickness
+    ℵ = sim.model.ice_concentration
+    u = sim.model.velocities.u
+    v = sim.model.velocities.v
+    push!(htimeseries, deepcopy(Array(interior(h))))
+    push!(ℵtimeseries, deepcopy(Array(interior(ℵ))))
+    push!(utimeseries, deepcopy(Array(interior(u))))
+    push!(vtimeseries, deepcopy(Array(interior(v))))
+end
+
+wall_time = [time_ns()]
+
+function progress(sim) 
+    h = sim.model.ice_thickness
+    ℵ = sim.model.ice_concentration
+    u = sim.model.velocities.u
+    v = sim.model.velocities.v
+
+    hmax = maximum(interior(h))
+    ℵmin = minimum(interior(ℵ))
+    umax = maximum(interior(u)), maximum(interior(v))
+    step_time = 1e-9 * (time_ns() - wall_time[1])
+
+    @info @sprintf("Time: %s, Iteration %d, Δt %s, max(vel): (%.2e, %.2e), max(trac): %.2f, %.2f, wtime: %s \n",
+                   prettytime(sim.model.clock.time),
+                   sim.model.clock.iteration,
+                   prettytime(sim.Δt),
+                   umax..., hmax, ℵmin, prettytime(step_time))
+
+     wall_time[1] = time_ns()
```

Comment:
```suggestion
     wall_time[] = time_ns()
```

Reply:
fi you put the "Array pattenrn" in an example, then users will start copying it.